visual-testing-app/webpack.config.js

```diff
@@ -75,7 +75,7 @@ module.exports = {
     },
   },
   plugins: [
-    new CleanWebpackPlugin(['dist']),
+    new CleanWebpackPlugin(),
```

Comment:
[ref](https://github.com/johnagan/clean-webpack-plugin/issues/106)

Reply:
Yeah should work for UIKit in our apps (and hence app-kit) the change is a bit more involved.